addons/account/views/account_invoice_view.xml

```diff
@@ -148,6 +148,18 @@
             </field>
         </record>
 
+        <record id="invoice_tree_with_onboarding" model="ir.ui.view">
+            <field name="name">account.invoice.tree.with.onboarding</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="invoice_tree"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//tree" position="attributes">
```

Comment:
Is there an action that doesnt need this ? If yes then ok.

Reply:
The "Customer credit notes" action.